aspnetcore/fundamentals/minimal-apis/samples/WebMinAPIs/Program.cs

```diff
@@ -813,4 +813,20 @@ class Product
 app.Run();
 #endregion
 
+#elif FILEUPLOAD
+#region snippet_fileupload
+var builder = WebApplication.CreateBuilder(args);
+var app = builder.Build();
+
+app.MapPost("/uploadfile", async (IConfiguration config, HttpRequest request) =>
```

Comment:
```suggestion
app.MapPost("/uploadstream", async (IConfiguration config, HttpRequest request) =>
```

I think "/uploadstream" might be a better name long term since we are planning on adding `IFormFile` support to minimal APIs in .NET 7.